src/components/Search/Search.module.scss

```diff
@@ -1,8 +1,8 @@
 .container {
-  height: 100vh;
-  display: flex;
-  align-items: center;
-  justify-content: center;
+  position: fixed;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
```

Comment:
hmm, I dont get it. what's the point?

Reply:
Okey, I've centered it with flexbox and had to do few other changes